refinery/core/admin.py

```diff
@@ -9,10 +9,14 @@
 from django_extensions.admin import ForeignKeyAutocompleteAdmin
 from guardian.admin import GuardedModelAdmin
 
-from .models import (Analysis, AnalysisNodeConnection, AnalysisResult, DataSet,
-                     Download, ExtendedGroup, InvestigationLink, Invitation,
-                     Ontology, Project, SiteProfile, Tutorials, UserProfile,
-                     Workflow, WorkflowEngine)
+from tool_manager.utils import AdminFieldPopulator
+
+from .models import (
+    Analysis, AnalysisNodeConnection, AnalysisResult, DataSet, Download,
+    ExtendedGroup, InvestigationLink, Invitation, Ontology, Project,
+    SiteProfile, SiteStatistics, Tutorials, UserProfile, Workflow,
+    WorkflowEngine
+)
```

Comment:
Unnecessary line breaks

Reply:
Regarding all of your comments about unnecessary line breaks/reformatting of import statements: the new isort config does this automatically. [This configuration](https://github.com/refinery-platform/refinery-platform/blob/develop/.isort.cfg#L27) was the closest I could find that satisfied your concerns from [here](https://github.com/refinery-platform/refinery-platform/pull/2646#discussion_r174910995)

If this is a big deal then it should be brought up with the team and we should make a decision on it.

Reply:
The decision to apply new formatting rules was made without consulting the team, so that's the conversation that needs to happen first.

Reply:
At the last attended dev meeting, I thought we concluded we didn't care which as long as we picked one and stuck with it.

Reply:
This is different. Ilya commented on a PR of mine [here](https://github.com/refinery-platform/refinery-platform/pull/2646#discussion_r174910995) regarding the styling of import statements. I took this as blocking for that PR, and implemented the closest [`isort` configuration](https://github.com/refinery-platform/refinery-platform/blob/develop/.isort.cfg#L27) that I could find that addressed his concerns.

Reply:
Right, there was nothing decided. I don't think applying new formatting rules is a good idea just because a tool insists on doing so. My vote is to follow existing de facto style and avoid extra line breaks.